src/HumanView.cpp

```diff
@@ -137,3 +380,5 @@ void HumanView::checkKeyboard(float time) {
         display.close();
```

Comment:
Remove this Escape key pressed and move to the end of the game state switch in update